.gitignore

```diff
@@ -1,5 +1,5 @@
+lib-test
 dist
-lib
```

Comment:
How about we discuss this tomorrow at our meeting and figure out a plan. We have also been thinking it would be better to use master instead of vy2 at some point. But we can coordinate when we want to do this.